manual/manual/refman/classes.etex

```diff
@@ -395,6 +398,21 @@ method-name ':' poly-typexpr@.  It specifies whether the method is
 public or private, and gives its type. If the method is intended to be
 polymorphic, the type must be explicitly polymorphic.
 
+\subsubsection*{Explicit overriding}
+
+Since Ocaml 3.12, the keywords @"inherit!"@, @"val!"@ and @"method!"@
+have the same semantics as @"inherit"@, @"val"@ and @"method"@, but
+they additionally require the definition they introduce to be an
+overriding. Namely, @"method!"@ requires @method-name@ to be already
```

Comment:
Good point, the current wording also makes me stop at some point, droping the "an" sounds better.